libraries/botbuilder-dialogs/src/memory/scopes/turnMemoryScope.ts

```diff
@@ -12,7 +12,7 @@ import { DialogContext } from '../../dialogContext';
 /**
  * @private
  */
-const TURN_STATE = Symbol('turn');
+const TURN_STATE = 'turn';
```

Comment:
@stevengum, the only issue I could see here is conflicting with code that used the `'turn'` state key. Is that me being too paranoid about backward compatibility?

Reply:
We do have some built in turn states with string keys, is there a way to notify our users that these keys are _reserved_ keys?

Reply:
Not that I know of - @stevengum any thoughts?